rest/taskrouter/activities/instance/get/example-1/example-1.5.x.rb

```diff
@@ -0,0 +1,14 @@
+# Get twilio-ruby from twilio.com/docs/ruby/install
+require 'rubygems'          # This line not needed for ruby > 1.8
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from twilio.com/user/account
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+workspace_sid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+activity_sid = 'WAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+
+client = Twilio::REST::Client.new account_sid, auth_token
+
+activity = client.taskrouter.v1.workspaces(workspace_sid).activities(activity_sid).fetch()
```

Comment:
This one refers to the `fetch` method, right? In that ok with removing the parens here. _Just for `fetch`.